managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -124,6 +124,17 @@ public UniverseController(YBClientService service) {
     this.ybService = service;
   }
 
+  private boolean validateEncryption(ObjectNode formData) {
+    boolean returnValue = true;
+    for (JsonNode cluster : formData.get("clusters")) {
+      JsonNode nodeToNodeEncryption = cluster.get("userIntent").get("enableNodeToNodeEncrypt");
+      JsonNode clientToNodeEncryption = cluster.get("userIntent").get("enableClientToNodeEncrypt");
+
+      returnValue = returnValue && (!nodeToNodeEncryption.asBoolean() && clientToNodeEncryption.asBoolean());
+    }
```

Comment:
Despite the previous comment, this one is about the loop - you can exit the loop if the variable becomes false.

Reply:
Done